phoenix-core/src/main/java/org/apache/phoenix/query/GuidePostsCache.java

```diff
@@ -43,13 +43,13 @@
 import org.slf4j.LoggerFactory;
 
 import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
 import com.google.common.cache.LoadingCache;
 import com.google.common.cache.RemovalCause;
 import com.google.common.cache.RemovalListener;
 import com.google.common.cache.RemovalNotification;
 import com.google.common.cache.Weigher;
 
+
```

Comment:
The GuidePostsCache was mostly rewritten here.  Is it possible to add a unit test for this?

Reply:
In the current code base, The GuidePostsCache doesn't have unit test except the inner class PhoenixStatsCacheRemovealListner. My change DOES NOT rewrite GuidePostsCache that much. As you can see, my main changes in GuidePostsCache are: a. initialize ExecutorService object and initialize Guava cache object with refresh mechanism; b. trivial changes in the inner class StatsLoaderImpl. I didn't change the interface of GuidePostsCaches including get/put/invalidate* APIs. My main changes are in PhoenixStatsCacheLoader class which I added unit test. Adding unit test for GuidePostsCache shouldn't be addressed by this change.